controllers/user-controller.js

```diff
@@ -0,0 +1,50 @@
+const userService = require('../service/user-service');
+const {validationResult} = require('express-validator');
+const ApiError = require('../api-error');
+
+class UserController {
+    async register(req, res, next) {
+        try {
+            const errors = validationResult(req);
+
+            if(!errors.isEmpty()){
+                return next(ApiError.BadRequest("Ошибка при валидации", errors.array()))
+            }
+
+            const {email, password} = req.body;
+            const userData = await userService.register(email, password);
+
+            res.cookie('refreshToken', userData.refreshToken, {maxAge: 30 * 24 * 60 * 60 * 1000, httpOnly: true});
+
+            return res.json(userData);
+        } catch (e) {
+            next(e);
+        }
+    }
+
+    async login(req, res, next) {
+        try {
+            const {email, password} = req.body;
+            const userData = await userService.login(email, password);
+            res.cookie('refreshToken', userData.refreshToken, {maxAge: 30 * 24 * 60 * 60 * 1000, httpOnly: true});
+
+            return res.json(userData);
+        } catch (e) {
+            next(e);
+        }
+    }
+    
+    async refresh(req, res, next){
+        try{
+            const {refreshToken} = req.cookies;
+            const userData = await userService.refresh(refreshToken);
+
+            res.cookie('refreshToken', userData.refreshToken, {maxAge: 30 * 24 * 60 * 60 * 1000, httpOnly: true});
+            return res.json(userData)
+        } catch (e) {
+            next(e);
+        }
+    }
+}
+
+module.exports = new UserController();
```

Comment:
The same export of the single class instance. Please export an object or a class itself.